dbt_subprojects/hourly_spellbook/models/_project/cow_protocol/cow_protocol_batches.sql

```diff
@@ -51,26 +51,4 @@ FROM
             unwraps,
             token_approvals
         FROM {{ ref('cow_protocol_gnosis_batches') }}
-
-         UNION ALL
-
-        SELECT
-            'arbitrum' AS blockchain,
-            'cow_protocol' AS project,
-            '1' AS version,
-            block_date,
-            block_time,
-            num_trades,
-            dex_swaps,
-            batch_value,
-            solver_address,
-            tx_hash,
-            gas_price,
-            gas_used,
-            tx_cost_usd,
-            fee_value,
-            call_data_size,
-            unwraps,
-            token_approvals
-        FROM {{ ref('cow_protocol_arbitrum_batches') }}
```

Comment:
Why remove this union?

Reply:
Great catch (I wanted to remove some base references I had locally, but went too far)

<img src="https://media0.giphy.com/media/2k5geABYZsReo/giphy.gif"/>

Reply:
hitting home for me on this one, i'm a reds fan!